medmodels/treatment_effect/estimate.py

```diff
@@ -532,9 +527,9 @@ def hazard_ratio(self, medrecord: MedRecord) -> float:
             number_control_outcome_true + number_control_outcome_false
         )
 
-        if hazard_control == 0:
-            msg = "Control hazard rate is zero, cannot calculate hazard ratio."
-            raise ValueError(msg)
+        if hazard_control == 0:  # pragma: no cover
+            msg = "Should not be reached"
+            raise NotImplementedError(msg)
```

Comment:
"should never be reached" checks are done due to typing. If this actually can never be 0, this check is not necessary. a comment above `return hazard_treat / hazard_control` saying that `hazard_control` cannot be 0 is sufficient